common/include/pcl/conversions.h

```diff
@@ -205,10 +205,10 @@ namespace pcl
     else
     {
       // If not, memcpy each group of contiguous fields separately
-      for (std::uint32_t row = 0; row < msg.height; ++row)
+      for (std::size_t row = 0; row < msg.height; ++row)
```

Comment:
Should this be replaced with `index_t` (or `uindex_t`)?

Reply:
Changed to index_t.

Reply:
That was a question to maintainers :)